packages/relay/src/lib/eth.ts

```diff
@@ -1720,9 +1721,16 @@ export class EthImpl implements Eth {
     if (submittedTransactionId) {
       try {
         const formattedTransactionId = formatTransactionIdWithoutQueryParams(submittedTransactionId);
+
+        // Create a modified copy of requestDetails
+        const modifiedRequestDetails = {
```

Comment:
This works but is there a place where we can do this once before it's passed into the eth.ts but after it's utilized locally?

Reply:
Well, I haven't come up with anything, cause actually we use the ip address in the shouldLimit method, the flow is the following
sendRawTransaction -> submitEthereumTransaction -> executeTransaction -> shouldLimit
So if i change it before sendRawTransaction a.k.a in eth.ts it will be changed for shouldLimit as well
Also I did it in the fastest way possible without affecting a lot of tests, because i thought it should go to mainnet asap